Lib/sqlite3/__init__.py

```diff
@@ -21,3 +21,17 @@
 # 3. This notice may not be removed or altered from any source distribution.
 
 from sqlite3.dbapi2 import *
+
+
+# OptimizedUnicode was deprecated in Python 3.10.  It's scheduled for removal
```

Comment:
Please add a reference to the bpo-42264:

```suggestion
# bpo-42264: OptimizedUnicode was deprecated in Python 3.10.  It's scheduled for removal
```

I let you reformat the comment if needed.

Reply:
Good idea. Fixed in fe7c65e76213f9b4565ef3eba37008e60f2d834f